### PR TITLE
Add cronjob alerts

### PR DESCRIPTION
**Pull request by palson-cf**

### CODE REVIEW

File: alerts/dind.yaml [hunk 1]
```diff
@@ -347,3 +347,47 @@
...
+    summary: CronJob *{{ $labels.label_cronjob}}* has been failing for the last hour
+
+- alert: KubeCronJobNotRunning
+  expr: (time() - kube_cronjob_status_last_schedule_time) > bool 262800 == 1
+  for: 1h
+  labels:
+    severity: critical
+    alertgroup: downtime
+  annotations:
+    message: CronJob *{{ $labels.label_cronjob}}* has not been running for the last three days
```
Comment (sharon-codefresh) [inline alerts/dind.yaml:392]: 3 days it's not too much?

File: alerts/dind.yaml [hunk 2]
```diff
@@ -347,3 +347,47 @@
...
+      group_right()
+      kube_job_labels{label_cronjob!=""}
+    ) by (job_name, label_cronjob)
+    == on(label_cronjob)
+    group_left()
+    max(
+      kube_job_status_start_time
+      * on (job_name)
+      group_right()
+      kube_job_labels{label_cronjob!=""}
```
Comment (sharon-codefresh) [inline alerts/dind.yaml:365]: why it's important to have this label?

File: alerts/dind.yaml (hunk 1 above)
Comment (palson-cf) [reply alerts/dind.yaml:392]: We have a job with the schedule
` "5,15,25,35,45,55 * * * 2-5"`

File: alerts/dind.yaml (hunk 2 above)
Comment (palson-cf) [reply alerts/dind.yaml:365]: To group jobs by cronjob

**Review by vadimgusev-codefresh — APPROVED** [review]

**Pull request merged**